source/billing.txt

```diff
@@ -173,18 +178,18 @@ and increases by 1 for every 32MB of memory that a given request uses.
 
 **Formula:** (# Requests) * (Runtime (ms)) * (Memory (MB) / 32MB) * $0.000000005 / ms
```

Comment:
I know you didn't add this, but the # here could be confusing. Not everyone will interpret this to mean "number of". Can we change this here and in line 216 where it has a similar construction? Should adhere to our placeholder value guidelines. The [API placeholder guidelines](https://www.mongodb.com/docs/meta/style-guide/style/cloud-account-info/) are probably most relevant.

```suggestion
**Formula:** (numberOfRequests) * (Runtime (ms)) * (Memory (MB) / 32MB) * $0.000000005 / ms
```

Reply:
What do you think about leaving it as more of a word phrase than a code placeholder? Think this page is more focused on being human readable rather than translatable to code.